manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -109,22 +101,18 @@ spec:
         severity: warning
     - alert: etcdBackendQuotaLowSpace
       annotations:
-        summary: etcd cluster database size exceeds the defined quota.
-        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the
-          defined quota on etcd instance {{ $labels.instance }}, please defrag or
-          increase the quota as the writes to etcd will be disabled when it is full.'
+        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the defined quota on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.'
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdBackendQuotaLowSpace.md
+        summary: etcd cluster database is running full.
       expr: |
         (etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100 > 95
       for: 10m
       labels:
         severity: critical
     - alert: etcdExcessiveDatabaseGrowth
       annotations:
-        summary: etcd cluster database size increased by 50 percent over the past four hours.
-        description: 'etcd cluster "{{ $labels.job }}": Observed surge in etcd writes
-          leading to 50% increase in database size over the past four hours on etcd
-          instance {{ $labels.instance }}, please check as it might be disruptive.'
+        description: 'etcd cluster "{{ $labels.job }}": Observed surge in etcd writes leading to 50% increase in database size over the past four hours on etcd instance {{ $labels.instance }}, please check as it might be disruptive.'
+        summary: etcd cluster database growing very fast.
       expr: |
         increase(((etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100)[240m:1m]) > 50
```

Comment:
unrelated nit, but putting `etcd_server_quota_backend_bytes` inside the `increase` could lead to false-positives if the user scales down the quota because they had overprovisioned.  What you care about is consumption vs. the current quota, so maybe:

    increase(etcd_mvcc_db_total_size_in_bytes[4h]) / etcd_server_quota_backend_bytes *100 > 50

with a `description` like:

> etcd cluster `"{{ $labels.job }}"`: etcd database size increased `{{ $value }}%` of the configured quota over the past four hours on etcd instance `{{ $labels.instance }}`.  Please defrag (FIXME: doc link?) or increase the quota (FIXME: doc link?) as the writes to etcd will be disabled when it is full.  Alternatively, investigate consumption and see if you can remove whatever's spewing this data into etcd (FIXME: reword.  Doc link).